jenkins-plugin/src/main/java/org/jenkinsci/plugins/pipeline/maven/GlobalPipelineMavenConfig.java

```diff
@@ -543,6 +543,7 @@ public FormValidation doValidateJdbcConnection(
                             }
                         }
                         switch (metaData.getDatabaseMajorVersion()) {
+                            case 14:
```

Comment:
PostgreSQL integration test is done on the "latest" tag of the Docker image, and right now it points to the 14.x branch